src/jcpsantiago/arqivist/api/confluence/handlers.clj

```diff
@@ -243,12 +241,39 @@
 
       (mulog/log ::lifecycle-event :event-type event-type :base-url base-url :local-time (java.time.LocalDateTime/now))
 
-      (mulog/with-context
-       {:base-url (:baseUrl lifecycle-payload)
-        :event-type event-type
-        :tenant-id tenant_id}
+      (mulog/with-context {:base-url (:baseUrl lifecycle-payload) :event-type event-type :tenant-id tenant_id}
+                          (case event-type
+                            "installed" (installed lifecycle-payload atlassian_tenant system)
+                            "enabled" (enabled lifecycle-payload atlassian_tenant system)
+                            "uninstalled" (uninstalled lifecycle-payload atlassian_tenant system))))))
+
+(defn get-started-handler
```

Comment:
I tend to drop handler in the name of a handler function, especially when the namespace and namespace alias is also handler..